docs/index.md

```diff
@@ -2,32 +2,43 @@
 
 <p style="font-size: larger; font-weight: bold;">Transform your research with vast amounts of computing!</p>
 
-The OSG Virtual School 2021 will prepare you to use high throughput computing (HTC) systems&nbsp;&mdash;
-locally or using the OSG&nbsp;&mdash;
+The OSG Virtual School 2021 will prepare you to use high throughput computing (HTC)&nbsp;&mdash;
+whether on a 'local' system or using the 
+[Open Science Grid (OSG)](https://www.youtube.com/watch?v=t2PzIy-vvaE&list=PLBWb4iScSWcPy7LQ4BuXmm8Z2xO4ZW1J_)&nbsp;&mdash;
 to run large-scale research simulation, analysis, and other applications for cutting-edge science.
-Through lecture, demonstration, hands-on exercises, and one-on-one consulting with experienced OSG staff,
+Through lecture, demonstration, hands-on exercises, and peronalized consulting with experienced OSG staff,
 you will learn to harness HTC systems and apply them to your own research computing work.
-The event has been tuned for remote participation and is not merely a translation of the in-person program!
-
-The Virtual School will be held **the weeks of August 2&ndash;6 and 9&ndash;13**.
-You can view [the high-level schedule](schedule.md) now.
-
-Participation in the event is free, but a selective application process is required.
-The School is ideal for graduate students in any research domain
-where large-scale computing is a vital part of the research process,
-plus we will consider applications from advanced undergraduates, post-doctoral students, faculty, and staff.
-While no travel is involved and the schedule allows for some flexibility,
-the expectation is that participants commit to about 20 hours each week.
-
-**Applications will open in late April or early May;** check back here then.
-
-Note: We hope to return to an in-person OSG User School in 2022!
+The School is ideal for graduate students in any research area for whom beyond-the-desktop computing 
+is a vital part of the research process; we also accept applications from advanced undergraduates, 
+post-doctoral students, faculty, and staff!
+
+_We hope to return to an in-person OSG User School in 2022!_ This virtual offering has been tuned for remote 
+participation, with the goal of helping attendees get their own work running within a two-week period, and 
+is not merely a translation of the in-person program (last offered in [2019](https://opensciencegrid.org/user-school-2019/)).
+
+**The Virtual School will be held the weeks of August 2&ndash;6 and 9&ndash;13**. 
+Participation is **free** and with materials publicly posted. A selective application process is required 
+for live participation, which includes access to HTC systems via the 
+[Center for High Throughput Computing](https://chtc.cs.wisc.edu/approach.shtml) (at UW-Madison)
```

Comment:
First mention, so “University of Wisconsin&ndash;Madison”. Or more simply, omit the parenthetical, since that information can be determined via the link and is not critical to the message.